src/itest/resources/data/managing_officer_delta_out.json

```diff
@@ -5,8 +5,10 @@
       "service_address":{
         "address_line_1":"1 Crown Way",
         "address_line_2":"Pavement",
+        "care_of": null,
```

Comment:
Can these `null values just be left out? When the files are serialised missing fields should resolve to being null anyway

Reply:
Nulls are required for the tests to work, seems to do some strange matching logic within the OfficerRequestMatcher class.